test/test_autotokenizer.py

```diff
@@ -30,15 +30,15 @@ def test_llama_tokenizer(self):
 
     def test_falcon_tokenizer(self):
         # replace the official model name after the model is not gated anymore
-        tokenizer = AutoTokenizer.from_pretrained("hf-internal-testing/llama-tokenizer")
+        tokenizer = AutoTokenizer.from_pretrained("Rocketknight1/falcon-rw-1b")
```

Comment:
```suggestion
        tokenizer = AutoTokenizer.from_pretrained("Rocketknight1/falcon-rw-1b", use_fast=False)
```

Using this way to get a slow Tokenizer, https://github.com/huggingface/transformers/blob/5e5fa0d88c293e6d5be2517b4f45680ba3bb5df2/src/transformers/models/auto/tokenization_auto.py#L594C17-L594C17

Reply:
perfect, missed this, thanks!